src/navigator/NavigationService.ts

```diff
@@ -231,6 +231,12 @@ export function navigateHome(fromModal?: boolean) {
   }, timeout)
 }
 
+export function navigateToSettings(promptConfirmRemovalModal: boolean) {
+  getFeatureGate(StatsigFeatureGates.USE_TAB_NAVIGATOR)
+    ? navigateClearingStack(Screens.Settings, { promptConfirmRemovalModal })
```

Comment:
actually would the user get stuck on the settings screen here? this would reset settings to the first screen. wonder if we should just navigate home, although not sure if it will break stuff. Maybe good to check in with product as well

Reply:
yeah I was thinking of just navigating to home, I'll check with product and see

Reply:
I synced with Laura on this and we don't want to navigate home instead of settings in most places. Seems like there's only a handful of places where we navigate to settings. So let's do this:
- change keyless backup progress screen to navigate home instead of settings (as the flow can start from either settings or notifications screen)
- to solve the edge case of feature gate changing when the user is on the Settings screen and backgrounded, I think we can get away with using the same screen enum. If we do that, then the version of settings screen that the flow started with will already be on the stack, so any `navigate(Screen.Settings)` would just pop back to that screen, instead of adding the new version of the screen to the stack. This should work as long as there is no flow that starts from somewhere else in the app and ends in settings, which I believe is only the keyless backup progress screen which we've addressed above.